backend/medtagger/repositories/users.py

```diff
@@ -43,3 +44,11 @@ def set_user_info(user: User, firstName: str, lastName: str) -> None:
             user.first_name = firstName
             user.last_name = lastName
             session.add(user)
+
+    @staticmethod
+    def set_user_settings(name: str, value: object) -> None:
+        """Set user's settings parameter of specified name to provided value."""
+        with db_session() as session:
+            settings = get_current_user().settings
```

Comment:
To overcome issue with using `get_current_user()`, please pass User object as an argument to this method.